app/views/metrics/index.html.erb

```diff
@@ -6,3 +6,130 @@
+		        <div class="menu">
+		          <div class="item" data-value="1">1</div>
+		          <div class="item" data-value="2">2</div>
+		          <div class="item" data-value="3">3</div>
+		        </div>
+		  	  </div>
+	  	  </span>
+	  	  late days by
+	  	  <span>
+		  	  <div class="ui selection dropdown">
+		        <input type="hidden" name="date">
+		        <i class="dropdown icon"></i>
+		        <div class="default text">Date</div>
+		        <div class="menu">
+		          <div class="item" data-value="1">Day 1</div>
+		          <div class="item" data-value="2">Day 2</div>
+		        </div>
+		  	  </div>
+	  	  </span>
+		</div>
+
+		<div class="metric">
+		<div class="inline field">
+		  <div class="ui checkbox">
+		    <input type="checkbox" tabindex="0" class="hidden">
+		  </div>
+	      Students whose grades have dropped by
+	      <span>
+		    <input type="text" placeholder="%" name="percentage">
+	  	  </span>
+	  	  percent within
+	  	  <span>
+		  	  <div class="ui selection dropdown">
+		        <input type="hidden" name="number">
+		        <i class="dropdown icon"></i>
+		        <div class="default text">#</div>
+		        <div class="menu">
+		          <div class="item" data-value="2">2</div>
+		          <div class="item" data-value="3">3</div>
+		          <div class="item" data-value="4">4</div>
+		        </div>
+		  	  </div>
+	  	  </span>
+	  	  consecutive assignments
+		</div>
+
+		<div class="metric">
+		<div class="inline field">
+		  <div class="ui checkbox">
+		    <input type="checkbox" tabindex="0" class="hidden">
+		  </div>
+	      Students who did not submit
+	      <span>
+		      <div class="ui selection dropdown">
+		        <input type="hidden" name="number">
+		        <i class="dropdown icon"></i>
+		        <div class="default text">#</div>
+		        <div class="menu">
+		          <div class="item" data-value="1">1</div>
+		          <div class="item" data-value="2">2</div>
+		          <div class="item" data-value="3">3</div>
+		        </div>
+		  	  </div>
+	  	  </span>
+	  	  assignments
+		</div>
+
+		<div class="metric">
+		<div class="inline field">
+		  <div class="ui checkbox">
+		    <input type="checkbox" tabindex="0" class="hidden">
+		  </div>
+	      Students with
+	      <span>
+		      <div class="ui selection dropdown">
+		        <input type="hidden" name="number">
+		        <i class="dropdown icon"></i>
+		        <div class="default text">#</div>
+		        <div class="menu">
+		          <div class="item" data-value="1">1</div>
+		          <div class="item" data-value="2">2</div>
+		          <div class="item" data-value="3">3</div>
+		        </div>
+		  	  </div>
+	  	  </span>
+	  	  submitted assignments below a percentage of
+	  	  <span>
+		    <input type="text" placeholder="%">
+	  	  </span>
+		</div>
+	</div>
+	<br>
+	<button class="ui submit button">SAVE</button>
+	<div class="ui error message"></div>
+  </div>
+
+</div>
```

Comment:
Missing some closing </div> tags. Footer is not appearing on the main page for that reason